contents/docs/experiments/creating-an-experiment.mdx

```diff
@@ -18,7 +18,9 @@ Here's a breakdown of each field in the form:
 
 ## Feature flag key
 
-Each experiment is backed by a [feature flag](/docs/feature-flags/manual). In your code, you use this feature flag key to check which experiment variant the user has been assigned to. 
+Each experiment is backed by a [feature flag](/docs/feature-flags/manual). In your code, you use this feature flag key to check which experiment variant the user has been assigned to.
+
+The feature flag will be created for you automatically. Therefore, you must add a feature flag key which does not exist yet.
```

Comment:
```suggestion
If the feature flag doesn't already exist, it will be created for you automatically when you create the experiment. ```

Reply:
Actually no - the feature flag with the given key _must not_ exist when you create the experiment.

Reply:
@jurajmajerik we have docs that say you can though? https://posthog.com/docs/experiments/common-questions#how-do-i-use-an-existing-feature-flag-in-an-experiment

Reply:
Oh wait misread. You are right. You need to delete the old one first